includes/save.php

```diff
@@ -298,6 +296,16 @@
         }
     }
 
+	// save timezone
+	if (!$update || strcmp($old_meeting->timezone, $_POST['timezone']) !==0) {
+		$changes[] = 'timezone'	;
+		if (empty($_POST['timezone'])) {
+			delete_post_meta($post->ID, 'timezone');
+		} else {
+			update_post_meta($post->ID, 'timezone', $_POST['timezone']);
```

Comment:
i think `timezone` needs to be sanitized. on line 47 we have an array of strings for sanitization, if you could add `timezone` to that array that should do it

Reply:
I thought about that, but because it's from a dropdown and cannot take text input from a user (unlike Time fields), I decided against it. Happy to change it if you still think it should be sanitized.

Reply:
yes, our policy is to sanitize every input that goes to the database to defend against sql injection, if you see any that we're not sanitizing let us know!

in this case it actually might be better to replace

```
		if (empty($_POST['timezone'])) {
```

with 
```
		if (!in_array($_POST['timezone'], DateTimeZone::listIdentifiers())) {
```

that way we can be extra sure we're getting the data we want